fedimint-rocksdb/src/lib.rs

```diff
@@ -1,30 +1,66 @@
-use super::batch::{BatchItem, DbBatch};
-use super::Database;
-use crate::db::PrefixIter;
 use anyhow::Result;
+use fedimint_api::db::batch::{BatchItem, DbBatch};
+use fedimint_api::db::Database;
+use fedimint_api::db::PrefixIter;
+use rocksdb::OptimisticTransactionDB;
+use std::path::Path;
 use tracing::{error, trace};
 
-impl Database for rocksdb::OptimisticTransactionDB {
+pub use rocksdb;
+
+#[derive(Debug)]
+pub struct RocksDb(rocksdb::OptimisticTransactionDB);
+
+impl RocksDb {
+    pub fn open(db_path: impl AsRef<Path>) -> Result<RocksDb, rocksdb::Error> {
+        let db: rocksdb::OptimisticTransactionDB =
+            rocksdb::OptimisticTransactionDB::<rocksdb::SingleThreaded>::open_default(&db_path)?;
+        Ok(RocksDb(db))
+    }
+
+    pub fn into_any(self) -> Box<dyn Database> {
```

Comment:
This (`_any`) is a bit of a misnomer, as `Any` and `dyn Trait` are a different trait.

Reply:
There's also a lot of `to_any` in the network stack. I should rename that too.

Reply:
Fixed in #626